### PR TITLE
Update compliance-easy-trials.md

### PR DESCRIPTION
**Pull request by kyracatwork**
Howdy! I'm a PM from the DLM & RM team. Please contact me at alias kyrachurney with any questions. 

Changes here are:
- Updating the first line of text for Records Management to be more descriptive & better match the flow of other solution wording.
- Updating the DLM section to include a first line of text and 2x bullet points to highlight features, to show more value to customers and better match the format/flow of the other solution wording.
- The terms & condition update is pushing it lower due to added content above.

### CODE REVIEW

File: microsoft-365/compliance/compliance-easy-trials.md [hunk 1]
```diff
@@ -97,8 +97,13 @@ Wondering what you can experience in your free trial? The Purview solutions tria
 
 - **Data Lifecycle Management**
 
-  Automate your retention coverage by using adaptive policy scopes. This feature allows you to dynamically target policies for retention to specific users, groups, or sites. These policies automatically update when changes occur in your organization. In addition, policies that use adaptive scopes aren't subject to location limits. Learn more about [adaptive policy scopes](retention.md#adaptive-or-static-policy-scopes-for-retention).
+  Microsoft Purview Data Lifecycle Management premium capabilities give you more automation and deeper customizations for how you manage the lifecycle of data – keeping what you need and getting rid of what you don’t need across the broad scale of your organization with:
+ 
+   - Adaptive policy scopes – govern data dynamically across your org by deploying retention policies and labels to groups of users, SharePoint sites, and Microsoft 365 Groups (including Microsoft Teams) using scopes that target policies to locations against attributes and properties that you define.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-easy-trials.md:102]:
```suggestion
   - Adaptive policy scopes – govern data dynamically across your organization by deploying retention policies and labels to groups of users, SharePoint sites, and Microsoft 365 Groups (including Microsoft Teams) using scopes that target policies to locations against attributes and properties that you define.
```

File: microsoft-365/compliance/compliance-easy-trials.md [hunk 2]
```diff
@@ -140,15 +145,15 @@ Wondering what you can experience in your free trial? The Purview solutions tria
...
+  Use integrated Microsoft Purview Records Management features to automate the retention schedule for organizational regulatory, legal, and business-critical records. Get full content lifecycle support, from creation to collaboration, record declaration, retention, and disposition:
 
   - Classify content as a record to prevent users from editing, as required by regulations, laws, or organizational policy
   - Apply retention labels to content automatically when it matches criteria you specify, using auto-apply label policies
   - Use adaptive scope policies to dynamically target your retention label policies to locations, with no limit on how many locations are included
   - Get full content lifecycle support, including the ability to perform disposition review on contents before they're permanently deleted at the end
 
-  For more information on the full range of feature for Microsoft Records Management, learn more about [Records Management](records-management.md)
+  For more information on the full range of features for Microsoft Purview Records Management, learn more about [Records Management](records-management.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-easy-trials.md:155]:
```suggestion
  For more information on the full range of features for Microsoft Purview Records Management, learn more about [Records Management](records-management.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kyracatwork, kindly commit the minor suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@kyracatwork Gentle reminder. Thanks

**Comment by LauraWi** [comment]
@chvukosw Can you take a look and merge if acceptable?

**Pull request merged**